central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl_create_req_test.go

```diff
@@ -193,10 +194,11 @@ func testApproval(t *testing.T) {
 	assert.NoError(t, err)
 
 	imageDataStore := imageDSMocks.NewMockDataStore(mockCtrl)
+	imageV2DataStore := imageV2DSMocks.NewMockDataStore(mockCtrl)
```

Comment:
You might have to change something where the test mocks the test to mock imageDataStore functions. It should mock imageV2DataStore functions when flag is set.